pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -63,6 +66,15 @@ func (enqueue *enqueueAction) Execute(ssn *framework.Session) {
 			}
 		}
 
+		// Reserve pg min resource in advance.
+		if (job.PodGroup.Status.Phase == scheduling.PodGroupInqueue ||
```

Comment:
let use swithc for `job.PodGroup.Status.Phase`.

Reply:
Phase is just part of the conditions here.

Reply:
check other condition in the `case`.